app/views/claims/claims.scss

```diff
@@ -1,5 +1,5 @@
 #trunk-content {
-  background-color:#d1c4c0;
+  background-color:#565656;
```

Comment:
unrelated styling fix while here. the text was a bit hard to read without more contrast